modules/cbQuestion/workflow/mview.php

```diff
@@ -32,7 +32,11 @@ function CBQuestionMViewFunction($entityData) {
 		list($void, $eid) = explode('x', $entityData->getId());
 		while ($cbq = $adb->fetch_array($qs)) {
 			$vname = str_replace(' ', '_', $cbq['qname']);
-			$adb->query('delete from '.$vname.' where '.$cbq['uniqueid'].'='.$eid);
+			if (!empty($cbq['maintablealias'])) {
+				$adb->query('delete from '.$vname.' where '.$cbq['maintablealias'].'.'.$cbq['uniqueid'].'='.$eid);
+			} else {
+				$adb->query('delete from '.$vname.' where '.$cbq['uniqueid'].'='.$eid);
+			}
```

Comment:
This is not needed as we are creating the whole SQL command and we know the name of the table we are deleting on.
undo this change, the maintablealias is only for the update SQL command we manipulate below.

Reply:
Okay